x-pack/plugins/task_manager/server/lib/create_managed_configuration.ts

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { interval, merge, of, Observable } from 'rxjs';
+import { filter, mergeScan, map, scan, distinctUntilChanged, startWith } from 'rxjs/operators';
+import { SavedObjectsErrorHelpers } from '../../../../../src/core/server';
+import { Logger } from '../types';
+
+const FLUSH_MARKER: string = 'FLUSH';
```

Comment:
super-nit: seems like a good use case for a Symbol - having said that, I've not used Symbols much, and not sure how good the TS support is for them.  

I'd also think that there must be some Rx thang that could deal with the flush logic, but I'm not familiar enough to suggest anything.

Reply:
Looks like TS didn't have any issues, it seems to work! Fixed in 
https://github.com/elastic/kibana/pull/77096/commits/17c44355d5bfc6533df48a3ad3b852a22b6144db.